interface/patient_tracker/patient_tracker.php

```diff
@@ -629,9 +645,14 @@ class="text-small"
                         <?php if ($GLOBALS['ptkr_show_encounter']) { ?>
                             <td class="detail text-center" name="kiosk_hide">
                                 <?php
+                                $signed = false;
                                 if ($appt_enc != 0) {
+                                    $signed = is_signed($appt_enc);
                                     echo text($appt_enc);
                                 }
+                                if ($signed) {
+                                    echo "<span class='text-success' title=''>&nbsp;&nbsp;<i class='fa fa-lock'></i></span>";
```

Comment:
For accessibility, I'd recommend adding a title here.

Reply:
@adunsulag the showing signer was not supposed to be part of this commit. That was a client request. I didn't even read what was bundled into this commit by mistake. I can clean it up and get it in. The client wanted to be able to see from the flow board which encounters had been signed or not. That way he can see what he needs to go and sign before end of his day. It was a feature in eCW.

Reply:
I have taken some time to think over what I was doing. I have made some code improvements. I moved the function out into the include file where is should be. Added better error handling and tightly coupled the return type.